README.md

```diff
@@ -28,6 +32,19 @@ The command line arguments below can be used with the command for each particula
 | -i, --input   | The graphql-configuration-file-path parameter specifies the path of the GraphQL config file (e.g., graphql.config.yaml) configured with GraphQL schemas (SDL) and GraphQL documents. This parameter is mandatory. You can point to the GraphQL config file by using the mandatory flag (-i,   --input).  |
 | -o, --output  | The output-location parameter specifies the path of the output location of the generated files.This parameter is optional. If this parameter is not specified, the Ballerina files will be generated at the same location from which the GraphQL command is executed. Optionally, you can point to another directory location by using the optional flag (-o, --output).  |
 
+### Command for GraphQL SDL schema generation
+
+The `graphql` command for GraphQL SDL schema generation and the usages are as follows.
+
+```
+bal graphql [-i | --input] <graphql-service-file-path> [-o | --output] <output-location> [-s | --service] <service-base-path>
+```
+
+| Argument      | Description   |
+| ------------- | ------------- |
+| -i, --input   | The graphql-service-file-path parameter specifies the path of the Ballerina GraphQL service file (e.g., service.bal). This parameter is mandatory. You can point to the Ballerina GraphQL service file by using the mandatory flag (-i,   --input).  |
```

Comment:
Unnecessary sentence? Also, how did this name `graphql-service-file-path parameter` occur here? I mean the flag and the parameter name has no connection, at the first glance. If we take the description out of the context, and read, how can we connect the `-i | --input` with `graphql-service-file-path parameter`? I think using the `the input parameter` is clearer.

```suggestion
| -i, --input   | The graphql-service-file-path parameter specifies the path of the Ballerina GraphQL service file (e.g., service.bal). This parameter is mandatory. 
```

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/pull/121/commits/33ace3fbc407810a3593b7b6758609bb9a608b42